lib/compress/fse_compress.c

```diff
@@ -321,19 +321,6 @@ size_t FSE_writeNCount (void* buffer, size_t bufferSize,
 /*-**************************************************************
 *  FSE Compression Code
 ****************************************************************/
```

Comment:
`FSE_sizeof_CTable()` seems purely removed, without replacement solution ?

Reply:
Yes, it isn't exposed in `fse.h`, and there are no callers that use a "hidden declaration". There is still the macro.